src/main/java/sqltoregex/deparser/ExpressionDeParserForRegEx.java

```diff
@@ -47,6 +47,7 @@ public ExpressionDeParserForRegEx(SelectVisitor selectVisitor, StringBuilder buf
     }
 
     protected void visitCommutativeBinaryExpression(BinaryExpression binaryExpression, String operator){
+        buffer.append("(?:");
```

Comment:
same discussion as in spellingmistake/order rotation ... capturing group optional?

Reply:
I don't think that it is necessary here, because you will never need to capture a expression. Its only needed for alias naming in the other generators.

Reply:
ok, fair enough